index.html

```diff
@@ -3,11 +3,63 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Form validation</title>
     <link rel="stylesheet" href="assets/css/style.css">
-    <script src="assets/js/app.js"></script>
+    <script defer src="assets/js/app.js"></script> 
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH" crossorigin="anonymous">
 </head>
-<body>
-    
+<body class="p-3 mb-2 bg-dark text-white">
+    <h1 class="text-center">Add Product</h1>
+    <hr>
+    <div class="container text-center">
```

Comment:
not so crucial for this mission, but please don't use the bootstrap grid system, rather native css `grid` + `flex`